auction_seller_invoice/security/auction_security.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="auction_seller_invoice_rule" model="ir.rule">
+        <field name="name">account.move Auction Seller Invoice</field>
+        <field name="model_id" ref="account.model_account_move" />
+        <field
+            name="domain_force"
+        >[('partner_id','=',user.commercial_partner_id.id)]</field>
+        <field
+            name="groups"
+            eval="[Command.link(ref('auction_seller.group_auction_seller'))]"
+        />
+    </record>
+    <record id="auction_seller_invoice_internal_user_rule" model="ir.rule">
+        <field name="name">account.move Auction Seller Invoice Internal User</field>
+        <field name="model_id" ref="account.model_account_move" />
+        <field name="domain_force">[(1,'=',1)]</field>
+        <field name="groups" eval="[Command.link(ref('base.group_user'))]" />
```

Comment:
```suggestion
        <field name="groups" eval="[Command.link(ref('acocung.group_account_invoice'))]" />
```
I think this would be more appropriate.